Duplicati/Library/Backend/TencentCOS/COSBackend.cs

```diff
@@ -0,0 +1,421 @@
+﻿using COSXML;
+using COSXML.Auth;
+using COSXML.Model.Bucket;
+using COSXML.Model.Object;
+using COSXML.Model.Tag;
+using COSXML.Utils;
+using Duplicati.Library.Common.IO;
+using Duplicati.Library.Interface;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Duplicati.Library.Backend
```

Comment:
The namespace should be `Duplicati.Library.Backend.TencentCOS` to be more consistent.